react-app-drawer-component/dist/styles.css

```diff
@@ -0,0 +1,59 @@
+.bar {
+  border: 2px solid black;
+  position: relative;
+  bottom: 20px;
+  background-color: white;
+  width: calc(100% / 3);
+  height: 100%;
+  font-family: sans-serif;
+  transition-property: all;
+}
+
+h1, h3 {
+  margin-left: 35px;
+}
+
+.open {
+  height: 100vh;
+  display: flex;
+  flex-direction: row;
+  transform: translateX(0);
+  transition-duration: 350ms;
+  z-index: 1;
+}
+
+.shade {
+  background-color: black;
+  width: 100%;
+  opacity: 0.5;
+  position: relative;
+  bottom: 20px;
+}
+
+.container {
+  height: 100vh;
+}
+
+.hidden {
+  transform: translateX(-280px);
+  transition-duration: 350ms;
+}
+
+.background-image {
+  height: 80%;
+  display: flex;
+  justify-content: center;
+  position: fixed;
+  top: 5;
+  right: 0;
+  bottom: 5;
+  left: 0;
+}
+
+.image {
+  border: 2px solid red;
+}
+
+.fa-star {
+  border: 2px solid black;
+}
```

Comment:
You don't want to directly add css rules to a class from a css library. This can cause issues in other parts of your project and/or confusion with other people working on the project. A better solution is to just make your own class and add it where you want it, for example:

```CSS
.icon-border {
  border: 2px solid black;
}
```

Reply:
tunrs out fa-star wasn't even existent as an element so it didnt do anything. I just deleted it